content/ember-4-0-released.md

```diff
@@ -0,0 +1,233 @@
+  the library [`@ember/legacy-built-in-components`](https://github.com/emberjs/ember-legacy-built-in-components)
+  as a stepping stone, following [this deprecation guide](https://deprecations.emberjs.com/v3.x#toc_ember-built-in-components-import).
+- ...and additional uncommon deprecations found in the [Ember.js 3.x deprecation
+  guide](https://deprecations.emberjs.com/v3.x)
+
+Many of these removed APIs date back to Ember 1.x, and are rarely used now (or should be rarely used).
+
+For more details on the changes in Ember.js 4.0, please review the
+[Ember.js 4.0.0 release page](https://github.com/emberjs/ember.js/releases/tag/v4.0.0).
+
+### Deprecations added in 4.0
+
+The following public API, marked as deprecated in Ember 4.0, will be removed in
+Ember 5.0:
+
+- Use of `Ember.assign` is deprecated. You should replace any calls to `Ember.assign` with `Object.assign` or use the object spread operator. See the [deprecation guide](https://deprecations.emberjs.com/v4.x#toc_ember-polyfills-deprecate-assign) for examples.
+
+## Changes in Ember Data 4.0
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+This release removes APIs that were [deprecated in the 3.x cycle](https://deprecations.emberjs.com/ember-data/v3.x). Examples include:
+
+- The API `store.defaultAdapter` is removed - [deprecation guide](https://deprecations.emberjs.com/ember-data/v3.x#toc_ember-data-default-adapter)
+- Support for relying on fallback behavior for adapter type is removed.
+  You should [specify adapter types explicitly](https://deprecations.emberjs.com/ember-data/v3.x#toc_ember-data-default-adapter).
+- Similarly, `adapter.defaultSerializer` and support for relying on fallback behavior for the
+  serializer type [is removed](https://deprecations.emberjs.com/ember-data/v3.x#toc_ember-data-default-serializers)
+- The [Evented API is removed](https://deprecations.emberjs.com/ember-data/v3.x#toc_evented-api-usage)
+- ...and other APIs listed in the [deprecation guide](https://deprecations.emberjs.com/ember-data/v3.x)
+
+For more details on the changes in Ember Data 4.0, please review the
+[Ember Data 4.0.0 release page](https://github.com/emberjs/data/releases/tag/v4.0.0).
+
+## Changes in Ember CLI 4.0
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+There are no deprecations, removals, or new features in Ember CLI v4.
+
+For more details on the changes in Ember CLI 4.0 and detailed upgrade
+instructions, please review the [Ember CLI 4.0.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v4.0.0).
+
+## Need help?
+
+Are you stuck on something? Do you have questions?
+Visit one of our [community chat groups](https://emberjs.com/community).
+
+## What next?
+
+Want to help make Ember 4 a success? Here are some things you can do!
+
+- Blog about your experience
+- If someone helps you with a question, consider positing both the question
```

Comment:
```suggestion
- If someone helps you with a question, consider posting both the question
```

Reply:
Oh wow, it seems `positing` is a word. Sorry if this suggestion incorrect!

Reply:
I think it's the incorrect use of the word anyway.